src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topBase;
+    private double bottomBase;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double topBase, double bottomBase,double height) {
+        super(color);
+        this.topBase = topBase;
+        this.bottomBase = bottomBase;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (topBase + bottomBase) / 2 * height;

```

Comment:
Consider storing the result of the area calculation in a local variable before returning it for better readability and potential debugging.